tests/arm64/corefx_test_exclusions.txt

```diff
@@ -0,0 +1,18 @@
+Invariant.Tests
+System.Diagnostics.Process.Tests
+System.IO.FileSystem.Tests
```

Comment:
This test takes ~ 30 minutes due to 6 tests which take ~5minute & 4 tests which take ~30s.
Also fails intermittently